spec/indexers/hyrax/valkyrie_collection_indexer_spec.rb

```diff
@@ -3,7 +3,12 @@
 require 'hyrax/specs/shared_specs'
 
 RSpec.describe Hyrax::ValkyrieCollectionIndexer do
-  subject(:indexer) { described_class.new(resource: resource) }
+  let(:resource_class) do
+    Class.new(Hyrax::Resource) do
+      # include ::Hyrax::CollectionBehavior # TODO: How is this represented in the Valkyrie::Resource version of a Collection?
```

Comment:
I could not find any example of a defined or generated Collection that is a Valkyrie::Resource.  The AF version includes CollectionBehavior.  I commented it out and left it here as a placeholder for any behaviors that are expected/required for a Valkyrie version of a Hyrax Collection.  I'd be interested in what others think should be included in this class.  NOTE: Prior to this PR, the Collection indexer was being tested with a Hyrax::Work as the resource passed to the indexer.

Reply:
Found it, this should be Hyrax::PcdmCollection